docs/technical-doc/api.rst

```diff
@@ -342,6 +342,34 @@ Get the block graph within the specified time interval.
         },
     ];
 
+`get_datastore_entry`
+--------------------
+
+Get a data entry both at the latest final and active executed slots.
+
+If an existing final entry is:
```

Comment:
For me it's unclear that you speak of the field `active_value` of the result type. I understand that you speak of the whole return type. Maybe use the field name : `final_value` could be more informative

```suggestion
If an existing final entry (`final_value`) is:
```